test/shopify_app/controller_concerns/login_protection_test.rb

```diff
@@ -173,6 +175,17 @@ class LoginProtectionTest < ActionController::TestCase
     end
   end
 
+  test '#fullpage_redirect_to skips rendering layout' do
+    LoginProtectionController.view_paths << ActionView::FixtureResolver.new(
```

Comment:
I'm not sure about this, because it will affect other tests since `LoginProtectionController.view_path` will not be reset between tests. How about something like: 0e18a43494df64ed5eb56c674ff16f59513f4ea1 ?

Reply:
I had no idea what `@_templates` is and why it returns such a hash, so I changed it slightly to make in more expressive what it refers to. What do you think?